docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+      - [Nonce-Commitment](#nonce-commitment)
+      - [MFA Cosigner Signature](#mfa-cosigner-signature)
+    - [GQ Signed Nonce-Commitment PK Tokens - Google Example](#gq-signed-nonce-commitment-pk-tokens---google-example)
+      - [GQ Signatures](#gq-signatures)
+      - [GQ Signature Protected Header](#gq-signature-protected-header)
+      - [GQ Signing](#gq-signing)
+    - [GQ Signed Audience-bound PK Tokens - GitHub Example](#gq-signed-audience-bound-pk-tokens---github-example)
+      - [Audience-Commitment](#audience-commitment)
+      - [GQ Signatures Are Required For Aud-Commitment PK Tokens](#gq-signatures-are-required-for-aud-commitment-pk-tokens)
+    - [GQ-Commitment PK Tokens (Gitlab-CI Example)](#gq-commitment-pk-tokens-gitlab-ci-example)
+      - [GQ-Commitment to the CIC](#gq-commitment-to-the-cic)
+      - [Security](#security)
+    - [ZK PK Tokens (Under development)](#zk-pk-tokens-under-development)
+  - [PK Token Compact Serialization](#pk-token-compact-serialization)
+    - [Refreshed Payload and Signature](#refreshed-payload-and-signature)
+- [Notes](#notes)
+  - [GQ Signatures](#gq-signatures-1)
+  - [Our JWS conventions](#our-jws-conventions)
+    - [The TYP pattern](#the-typ-pattern)
+    - [Protected header claims](#protected-header-claims-1)
+    - [Compact representations of a JWS with two or more signatures](#compact-representations-of-a-jws-with-two-or-more-signatures)
+
+<!-- /TOC -->
+
+
+
+## JSON Web Signatures (JWS) and JSON Web Tokens (JWT)
+
+A [JWS (JSON Web Signature)](https://www.rfc-editor.org/rfc/rfc7515.html) is a signed message format. The message which is signed is called the payload. It supports 1 or more signatures. Each signature has a protected header, denoted as `protected` in JSON, which specifies metadata about the signature such as the signing algorithm (`alg`) and the key ID (`kid`) of the public key which should be used to verify the signature.
+
+```json
+{"payload": "message payload"
+"signatures": [
+  {
+    "protected": {"alg": "RS256", "kid": "1234"},
+    "signature": "signature-1"
+  },
+  {
+    "protected": {"alg": "RS256", "kid": "5678"},
+    "signature": "signature-2"
+  },
+  {
+    "protected": {"alg": "RS256", "kid": "9123"},
+    "signature": "signature-3"
+  },
+]}
+```
+
+
+Note that each signature in a JWS signs the payload and the protected header associated with that signature. All signatures sign the same payload; no signature signs another signature's protected header. In the example above RSA signature-2 is computed as  `RSA-SIGN(SK, ("message payload", {"alg": "RS256", "kid": "1234"}))`.
+
+[JWT (JSON Web Token)](https://datatracker.ietf.org/doc/html/rfc7519) is a type of JWS used by one party, the issuer, to make claims another parties, the subject. The issuer includes their identity in the JWT using the `iss` claim. JWTs are defined as having only one signature, the signature of the issuer.
```

Comment:
Did you mean to say, "is a type of JWS used by one party, the issuer, to make claims **against** another party, the subject" ?

Reply:
Or maybe you meant "about" ?